client/gutenberg/editor/api-middleware/index.js

```diff
@@ -149,7 +156,7 @@ const wpcomProxyMiddleware = options => {
 		apiVersion,
 		apiNamespace = 'wp/v2',
 		onError = identity,
-		fromApi = identity,
+		fromApi = forceFeaturedImageSupport,
```

Comment:
This is more appropriate to pass in the wp/v2 mapping middleware, or a new middleware, if we're not able to make endpoint changes than to change the default here. We'll end up adding misc properties to unrelated endpoint responses.

We can also tidy code further once we have a few more examples of needing to pass along fromApi.
```jsx
 	if ( /\/wp\/v2\//.test( options.path ) ) {	 	if ( /\/wp\/v2\//.test( options.path ) ) {
 		const path = options.path.replace( '/wp/v2/', `/sites/${ siteSlug }/` );	 		const path = options.path.replace( '/wp/v2/', `/sites/${ siteSlug }/` );
 	 
    //if matches /themes
     const fromApi = isThemesEndpoint ? forceFeaturedImageSupport : identity;

 		return next( { ...options, fromApi, path, apiNamespace: 'wp/v2',  } );	 		return next( { ...options, path, apiNamespace: 'wp/v2' } );
 	}
```